src/WhatsAppCloudApi.php

```diff
@@ -51,6 +51,21 @@ public function __construct(array $config)
         $this->client = new Client($config['graph_version'], $config['client_handler']);
     }
 
+    protected ?string $contextMessageId = null;
+
+    /**
+     * Send the next message as a reply to the supplied message id
+     *
+     * @param $messageId
+     * @return $this
+     */
+    public function asReplyTo(string $messageId): WhatsAppCloudApi
```

Comment:
This superclass is not using Fluent API code style to simplify the code and isolate the responsibilities of the classes.